### PR TITLE
add Japanese chapter7 md1 and md2 second try.

### PR DESCRIPTION
**Pull request by webbigdata-jp**
Hi @lewtun .
This is second try. I add Japanese chapter7/md1, 2.
I recreated the environment.
I also have done git add before make style.

### CODE REVIEW

File: chapters/ja/chapter7/1.mdx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<FrameworkSwitchCourse {fw} />
+
+# イントロダクション
+
+[第3章](/course/chapter3)では、テキスト分類のためにモデルを微調整する方法を学びました。この章では、以下のような一般的な自然言語処理タスクに取り組みます。
```
Comment (lewtun) [inline chapters/ja/chapter7/1.mdx:5]: I think we'll need to put the language code here for the cross-link to work 

```suggestion
[第3章](/course/ja/chapter3)では、テキスト分類のためにモデルを微調整する方法を学びました。この章では、以下のような一般的な自然言語処理タスクに取り組みます。
```

**Review by lewtun — APPROVED** [review]
Thank you for this great translation @webbigdata-jp ! 

Apart from a small change in the cross-linking it looks good to me 🔥

File: chapters/ja/chapter7/1.mdx (hunk 1 above)
Comment (webbigdata-jp) [reply chapters/ja/chapter7/1.mdx:5]: @lewtun 
Yay! That's my first PR approval!

Thanks for checking.
I will include the link fix in the next pull request.

**Comment by lewtun** [comment]
@webbigdata-jp if I'm not mistaken, this PR is superceded by #267 right?

If yes, feel free to close this PR :)

**Comment by webbigdata-jp** [comment]
@lewtun 
Ya, you are right.
I should have closed it, thanks for checking.

**Pull request closed without merging**